ruff.toml

```diff
@@ -6,8 +6,25 @@ extend-select = [
 	"C901",
 	"PERF401",
 	"W",
+ 
+ 	# Ensure modern type annotation syntax and best practices
+	# Not including those covered by type-checkers or exclusive to Python 3.11+
+	"FA", # flake8-future-annotations
```

Comment:
Should comments follow the Python pattern of two spaces preceding a comment character? Or does toml have different best practices?

Reply:
I've been using https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml, backed by https://github.com/tamasfe/taplo for many months now. And I don't see an option to use double spaces: https://taplo.tamasfe.dev/configuration/formatter-options.html

The prettier toml plugin uses taplo as well: https://www.npmjs.com/package/prettier-plugin-toml

dprint's toml config also applies only one: https://dprint.dev/plugins/toml/config/